cmd/cmd_object.go

```diff
@@ -548,17 +556,35 @@ func getObject(ctx *cli.Context) error {
 		}
 	}
 
-	body, info, err := gnfdClient.GetObject(c, bucketName, objectName, opt)
-	if err != nil {
-		return toCmdErr(err)
-	}
+	if resumableDownload {
+		opt.PartSize = partSize
```

Comment:
Does  it need to determine whether partSize must be an integer of segment size ?

Reply:
if   partSize  <. segment || partSize % segment  != 0  return err